docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -213,15 +213,15 @@ for more information about the environment variables.
 
 ifndef::apm-server[]
 The index name to write events to. The default is
-+"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
++"{beatname_lc}-%{[beat_version_key]}-%\{+yyyy.MM.dd\}"+ (for example,
```

Comment:
This line should not be changed. This is documentation for other beats and not `apm-server`. Suggestion to revert to original code:
```suggestion
+"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
```

Reply:
The reason I also changed this is that beats changed the default from `beat.version` to `agent.version`. But the beats team probably needs to do a larger rework here for all their docs so it is fine with me to not change this here.